src/main/kotlin/org/rust/lang/core/psi/ext/RsItemsOwner.kt

```diff
@@ -174,17 +191,15 @@ fun RsItemsOwner.processExpandedItemsInternal(
 private fun RsElement.processItem(
     withMacroCalls: Boolean,
     recursionLimit: Int,
-    processor: (RsElement, Boolean) -> Boolean
+    processor: (RsElement) -> Boolean
 ): Boolean {
-    val existsAfterExpansionSelf = this !is RsDocAndAttributeOwner || evaluateCfg() != ThreeValuedLogic.False
-
     val derives: Sequence<RsMetaItem>? = if (this is RsAttrProcMacroOwner) {
```

Comment:
Is `attr.derives` always non-empty in `is ProcMacroAttribute.Derive -> attr.derives` branch? If yes we can add cfg check to that branch and remove cfg check in `derives.forEachIndexed`

Reply:
No, `attr.derives` can still be empty because the sequence is filtered lazily